src/runtime.rs

```diff
@@ -45,27 +40,9 @@ pub async fn set() -> Option<Runtime> {
                 }
             }
         }
-        Err(_) => {
-            error!("❌ unable to create a docker client on the host");
+        Err(err) => {
+            error!("❌ unable to create a docker client on the host: {}", err);
             return None;
         }
     }
 }
-
-fn get_podman_socket() -> Result<String> {
```

Comment:
I prefer the keeping the existing behavior of falling-back to the podman socket based on the default XDG location.  I think the existing UX is better than making a podman user provide a custom socket path (which they likely don't know) via a flag. The proposed UX is too unfriendly for podman users. 

We would need to branch somehow if the OS is windows though, to prevent XDG issues you saw when running dcp on windows

Reply:
Totally agree, this is what I had at the moment while I was figuring out a work around for XDG not being supported on windows.

Reply:
Alright, I reverted the commit out. I'll have a follow-up for #14 to fix that issue.